ApiReview/API.Polly.Core/NoDocs/Polly.Simmy/BehaviorChaosPipelineBuilderExtensions.cs

```diff
@@ -0,0 +1,14 @@
+// Assembly 'Polly.Core'
+
+using System;
+using System.Diagnostics.CodeAnalysis;
+using System.Threading.Tasks;
+using Polly.Simmy.Behavior;
+
+namespace Polly.Simmy;
+
+public static class BehaviorChaosPipelineBuilderExtensions
```

Comment:
This class name is a bit of a mouthful 😄

If we don't make it simpler, maybe `ChaosBehaviorPipelineBuilderExtensions` would be better?

Reply:
Actually, I think I'm gonna drop the `Chaos` prefix from the file name, the API is already clear as it is using the `Chaos` prefix: AddChaosBehavior, AddChaosLatency, AddChaosFault etc. 

Otherwise we would need to apply that change everywhere for consistency's sake, like:

* ChaosBehaviorPipelineBuilderExtensions 
* ChaosBehaviorActionArguments
* ChaosBehaviorConstants
* ChaosBehaviorStrategyOptions
* OnChaosBehaviorInjectedArguments

Same for other strategies, etc.